src/extraData.ts

```diff
@@ -0,0 +1,54 @@
+import { decode, encode } from '@msgpack/msgpack'
+
+export const MSG_PACK_ID = '0x544c4d50' // hex encoded "TLMP" (Trustlines Message Pack)
```

Comment:
Chance to save a comment. :shrug:

```suggestion
export const MSG_PACK_ID = binascii.hexlify(b"TLMP")
```

Reply:
If you have that function in javascript for me, I am all in.

Reply:
Oh dear :joy: